webrtc.html

```diff
@@ -8821,6 +8822,12 @@ <h3><dfn>RTCIceRole</dfn> Enum</h3>
             <tr>
               <th colspan="2"><code><a>RTCIceRole</a></code> Enumeration description</th>
             </tr>
+            <tr>
+              <td><dfn data-idl><code>unknown</code></dfn></td>
+              <td>An agent whose role as defined by [[!ICE]], Section 3,
+                has not yet been determined.
```

Comment:
(Is this the right way to format the td?)

Reply:
I wonder if "The role attribute MUST return the ICE role of the transport." is specified enough; if the role ever changes there needs to be an update step. E.g. as part of setting a description there could be a step to update the ice role internal slot. Care to file an issue for that? Non-blocker for this PR